scripts/apps/authoring/versioning/history/views/history.html

```diff
@@ -2,7 +2,7 @@
     <ul class="history-list">
         <li class="item"
             ng-class="{selected: selected === version}"
-            ng-repeat="version in versions track by version._current_version"
+            ng-repeat="version in versions"
```

Comment:
When we select different items to be previewed, some version revisions ( having same _current_version already on display) are not replaced. As we are now dealing with different items being selected for display, 'track by' was removed so that new item's versions are rendered on the DOM. Kindly suggest/correct me as if I'm missing anything.